app/uk/gov/hmrc/voapropertylinking/actions/AuthenticatedActionBuilder.scala

```diff
@@ -39,9 +42,9 @@ class AuthenticatedActionBuilder @Inject()(
       case externalId ~ groupIdentifier =>
         (externalId, groupIdentifier) match {
           case (Some(exId), Some(grpId)) => block(RequestWithPrincipal(request, Principal(exId, grpId)))
-          case (Some(_), None)           => Future.successful(Unauthorized(Json.obj("errorCode" -> "NO_GROUP_ID")))
-          case (None, Some(_))           => Future.successful(Unauthorized(Json.obj("errorCode" -> "NO_EXTERNAL_ID")))
-          case (None, None)              => Future.successful(Unauthorized(Json.obj("errorCode" -> "NO_EXTERNAL_ID_OR_GROUP_ID")))
+          case (Some(_), None) => Future.successful(Unauthorized(Json.obj("errorCode" -> "NO_GROUP_ID")))
```

Comment:
whats going on with the formatting?

Reply:
This project does not have scalafmt. I think this file was copy-paste from another service and never formatted by intellij. intellij uses its own formatting rules because there's no scalafmt for this codebase